dispatch/bin/dispatch-admin

```diff
@@ -71,15 +96,17 @@ commands = {
 
 if __name__ == "__main__":
 
-    command_name = sys.argv[1]
-    args = sys.argv[2:]
+    if len(sys.argv) == 1:
+        sys.argv.append("-h")
+
+    arguments = docopt(__doc__, version="Dispatch Admin 0.5.17")
```

Comment:
We can use the package's version instead of hard coding it

```suggestion
    arguments = docopt(__doc__, version="Dispatch Admin %s" % dispatch.__version__)
```